### PR TITLE
fix(theme):  修复颜色主题配置对自定义 icon 不生效的问题

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🐛 Bugfix

- [x] Solve the issue and close #0

### 📝 Description

icon 错误的使用了 文本颜色的主题配置, 导致 icon 颜色始终和文本一样

```ts
const s2Theme = {
  rowCell: {
    icon: {
      fill: 'red'
    }
  }
}
```

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| image |image|

### 🔍 Self Check before Merge

- [x] Add or update relevant Docs.
- [x] Add or update relevant Demos.
- [x] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/__tests__/spreadsheet/theme-spec.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      const actionIconCfg: ShapeAttrs = get(rowCell, 'actionIcons.[0].cfg');
+
+      expect(actionIconCfg.fill).toEqual(iconInfo.fill);
+      expect(actionIconCfg.name).toEqual(iconInfo.name);
+      expect(actionIconCfg.width).toEqual(iconInfo.size);
+      expect(actionIconCfg.height).toEqual(iconInfo.size);
+    });
+  });
+
+  describe('Row Header Icon Tests', () => {
```
Comment (lijinke666) [inline packages/s2-core/__tests__/spreadsheet/theme-spec.ts:79]: 这里开始是 packages/s2-core/__tests__/spreadsheet/icons-spec.ts 的测试, 统一移到 theme-spec

**Comment by xingwanying** [comment]
> ### 👀 PR includes
> 🐛 Bugfix
> 
> * [x]  Solve the issue and close #0
> 
> ### 📝 Description
> icon 错误的使用了 文本颜色的主题配置, 导致 icon 颜色始终和文本一样
> 
> ```ts
> const s2Theme = {
>   rowCell: {
>     icon: {
>       fill: 'red'
>     }
>   }
> }
> ```
> 
> ### 🖼️ Screenshot
> Before	After
> image	image
> ### 🔗 Related issue link
> ### 🔍 Self Check before Merge
> * [x]  Add or update relevant Docs.
> * [x]  Add or update relevant Demos.
> * [x]  Add or update relevant TypeScript definitions.

怎么看着好像还没对齐🤔

**Review by xingwanying — APPROVED** [review]

**Pull request merged**